spec/index.bs

```diff
@@ -16726,15 +16733,65 @@ expose real values whenever the feature is available on the adapter:
 - New WGSL extensions:
     - [=extension/subgroups=]
 
+<h3 id=texture-formats-tier1 data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"texture-formats-tier1"`
+</h3>
+
+Supports the below new {{GPUTextureFormat}}s with the {{GPUTextureUsage/RENDER_ATTACHMENT}},
+[=blendable=], `multisampling` capabilities and the {{GPUTextureUsage/STORAGE_BINDING}} capability
+with the {{GPUStorageTextureAccess/"read-only"}} and {{GPUStorageTextureAccess/"write-only"}}
+{{GPUStorageTextureAccess}}es:
+- {{GPUTextureFormat/"r16unorm"}}
+- {{GPUTextureFormat/"r16snorm"}}
+- {{GPUTextureFormat/"rg16unorm"}}
+- {{GPUTextureFormat/"rg16snorm"}}
+- {{GPUTextureFormat/"rgba16unorm"}}
+- {{GPUTextureFormat/"rgba16snorm"}}
+
+Allows the {{GPUTextureUsage/RENDER_ATTACHMENT}}, [=blendable=], `multisampling` and `resolve`
+capabilities on below {{GPUTextureFormat}}s:
+- {{GPUTextureFormat/"r8snorm"}}
+- {{GPUTextureFormat/"rg8snorm"}}
+- {{GPUTextureFormat/"rgba8snorm"}}
+
+Enables the extension {{GPUFeatureName/"rg11b10ufloat-renderable"}}.
```

Comment:
If we want this, it needs to be specified in the `a new device` algorithm. (Since these sections are not referenced from algorithms, we essentially treat these sections as non-normative. We should probably mark them as such.)

That said, I don't recall if the WG had agreement to do this, even though I think it was what we wanted. Can we remove this for now, update the table to say "rg11b10ufloat-renderable or texture-formats-tier1" for these cells, and then open a follow-up PR which:
- Updates `a new device`
- Re-simplifies the table
- Adds this note
- Also adds a note about this in the definition of `"rg11b10ufloat-renderable"` so it's discoverable by clicking the link to that feature from the format table

Reply:
Done

Reply:
Filed https://github.com/gpuweb/gpuweb/issues/5200.